io/EptReader.cpp

```diff
@@ -767,5 +775,88 @@ Dimension::Type EptReader::getCoercedTypeTest(const NL::json& j)
     return getCoercedType(j);
 }
 
+//============================================================================================
+// Pravin: APIs from here are written to make EPT Reader streamable.
+
+void EptReader::loadNextOverlap()
+{
+    Key key(m_keys.at(0));
+    m_keys.erase(m_keys.begin());
+    log()->get(LogLevel::Debug)
+        << "Streaming Data " << m_nodeId << "/" << m_overlaps.size() << ": "
+        << key.toString() << std::endl;
+
+    uint64_t startId(0);
+	m_pointTable.reset(new PointTable());
+	if (m_info->dataType() == EptInfo::DataType::Laszip)
```

Comment:
lasZip and Binary should be treated in the same way (just like in the run method)

Reply:
Done!!